lib/base_service.js

```diff
@@ -276,9 +276,11 @@ BaseService.prototype._requireModule = function(modName) {
     try {
         return P.resolve(require(modName));
     } catch (e) {
-        if (/^\//.test(opts.mod) || (opts.baseTried && opts.modsTried)) {
-            // we have a full path here which can't be required, or we tried
-            // all of the possible combinations, so bail out
+        if (/^\//.test(opts.mod) || (opts.baseTried && opts.modsTried) ||
+                e.message !== "Cannot find module '" + modName + "'") {
```

Comment:
I know that comparing the message is tricky, but:
- the message for that particular error has never changed thus far, making it unlikely to change
- you get the same code when, e.g., one of the dependencies doesn't have `node_modules` and is genuinely missing a module (or you've mistyped the module's name or something)

So I think this is the most pragmatic way of fixing the issue at hand.